ui/components/HomeComponents/ConnectionActions.js

```diff
@@ -1,26 +1,33 @@
-import { Button, Stack } from "@chakra-ui/react"
+import { Button, Stack, Flex } from "@chakra-ui/react"
 import { useRouter } from "next/router"
 import React from "react"
 
 const ConnectionActions = ({ service }) => {
   const router = useRouter()
 
   return (
-    <Stack direction="row" spacing={{ base: 1, sm: 2, md: 8 }} pt="30px">
+    <Flex direction={{ base: "column", md: "row" }} align="center" pt="30px">
```

Comment:
le alignItems n'a pas besoin d'être set ?

Reply:
https://chakra-ui.com/docs/components/flex#usage "alignItems is align"